Sources/TuistKit/Mappers/FocusTargetsGraphMappers.swift

```diff
@@ -19,6 +20,8 @@ public enum FocusTargetsGraphMappersError: FatalError, Equatable {
         switch self {
         case let .targetsNotFound(targets):
             return "The following targets were not found: \(targets.joined(separator: ", ")). Please, make sure they exist."
+        case .noTargetsFound:
+            return "No targets were found."
```

Comment:
I'd make this one more actionable:

```suggestion
            return "No targets were found. Ensure the query is valid query and matches targets in the graph."
```